src/web/components/Standfirst.tsx

```diff
@@ -42,6 +44,16 @@ const nestedStyles = css`
 	strong {
 		font-weight: bold;
 	}
+
+	[data-dcr-style='bullet'] {
+		display: inline-block;
+		content: '';
+		border-radius: 0.375rem;
+		height: 0.75rem;
+		width: 0.75rem;
+		margin-right: 0.125rem;
+		background-color: ${pillarPalette[theme].main};
```

Comment:
We have stopped using `pillarPalette` in favour of `palette` (which we build in `decidePalette`) - happy to pair on how we implement this!